src/naive.py

```diff
@@ -0,0 +1,99 @@
+"""Implementation of the naive exact matching algorithm."""
+
+################################################################
+# libraries:
+import sys
+
+################################################################
+# Functions:
+
+from align import get_edits
+from cigar import edits_to_cigar    
+
+def naive_algorithm(ref:str, read:str):
+    if ref == '' or ref == None:
+        return []
+    if read == '' or read == None:
+        return []
+    ref=ref.strip().upper()
+    read=read.strip().upper()
```

Comment:
Why are you doing this? I'm sure I never said that you could ignore case or whitespace... It will work, of course, since we only use DNA strings, but it limits the scope where your code can be used for no apparent reason.